locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,173 @@
+  }
+modal-active-number-of-exposures-part-three-premium = This chart does not include any exposures that are in-progress of being auto-removed. Once your exposures are fixed, they will be added to your total number of fixed exposures on the Fixed page.
+
+progress-card-auto-removed-headline = Auto-removed
+progress-card-in-progress-headline = In progress
+progress-card-locked-alt = Only available with { -brand-premium }
+
+progress-card-heres-what-we-fixed-headline-premium = Here’s what we fixed
+
+# Here’s What We Fixed Modal
+
+modal-heres-what-we-fixed-title = About what we fixed
+modal-heres-what-we-fixed-description-part-one = <b>Manually fixed</b> includes anything you’ve fixed yourself. All data breaches that require access to your accounts need
+  to be fixed manually, even if you’ve upgraded to { -brand-premium }.
+modal-heres-what-we-fixed-description-part-two = <b>Auto-removed</b> includes any exposures from data broker profiles that we’ve removed for you. This is available only for
+  { -brand-premium } subscribers.
+modal-heres-what-we-fixed-description-part-three = <b>In Progress</b> includes anything that we are currently
+  working on fixing. Removals typically take 7-14 days but the most
+  difficult sites could take longer. You may also start to see removals
+  happening within the same day.
+
+# About Exposure Types Modal
+
+modal-exposure-type-title = About exposure types
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be more than 1.
+modal-exposure-type-description = We search for you in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Here are the two types of exposures we find:
+modal-exposure-type-data-breach = <b>Data breach</b> means your information has been compromised in a breach and could be in the wrong hands.
+  Resolving these typically requires accessing your accounts, so you’ll need to take manual steps to resolve each breach even if you’ve upgraded to { -brand-premium }.
+modal-exposure-type-data-broker-part-one = <b>Info for sale</b> means a data broker site is publicly publishing and selling your personal info.
+  You’ll need to manually request removal from each site.
+modal-exposure-type-data-broker-part-two = If you’re a { -brand-premium } user, we automatically remove all profiles for you.
+  In both cases, removals typically take 7-14 days. Some can take longer, while others can happen within the hour.
+
+# About Exposure Statuses Modal
+
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan. It will always be plural.
+modal-exposure-status-description-premium = We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info.
```

Comment:
```suggestion
modal-exposure-status-description-premium = { $data_broker_sites_total_num ->
   *[other] We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Your exposures will have one of the following statuses:
}
```

Also removing the unnecessary line break.

Reply:
02f9fc56d